### PR TITLE
Main structure

### PR DESCRIPTION
**Pull request by NikitaBondarevv**

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -155,23 +209,603 @@ body {
...
+
+.portfolio li {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  cursor: pointer;
+}
+
+.portfolio li:hover {
+  background: url(image/portfolioImages/activeElementBackground.png) no-repeat 28px 0;
```
Comment (iliatcymbal) [inline styles.css]: why image? You should try it with bg gradient `background-image: linear-gradient(to right top, #5e00ca, #d5536f);`
see https://www.w3schools.com/css/css3_gradients.asp for more details

File: index.html [hunk 2]
```diff
@@ -16,7 +16,7 @@
     </div>
     <nav class="navigation">
       <ul class="nav-block">
```
Comment (iliatcymbal) [inline index.html]: if you have two navigations, then use `header-navigation` and `footer-navigation`,
no need class here `<ul class="nav-block">` 

CSS:
```
.header-navigation ul {}
...
.footer-navigation ul {}
```

File: index.html [hunk 3]
```diff
@@ -32,20 +32,250 @@
...
+                <figcaption>
+                  <span>IMAGE 4</span>
+                </figcaption>
+              </figure>
+            </li>
+          </ul>
           <button class="button-gallery"></button>
+          <button class="button-gallery-hidden"></button>
+        </div>
+        <ul class="nav-gallery">
```
Comment (iliatcymbal) [inline index.html]: `class="paging"`

File: index.html [hunk 4]
```diff
@@ -32,20 +32,250 @@
...
+              <figure>
+                <img src="image/10XImage.png" alt="Image">
+                <figcaption>
+                  <span>IMAGE 4</span>
+                </figcaption>
+              </figure>
+            </li>
+          </ul>
           <button class="button-gallery"></button>
+          <button class="button-gallery-hidden"></button>
```
Comment (iliatcymbal) [inline index.html]: `hidden` should be a separate class name `"button-gallery hidden"`

File: index.html [hunk 5]
```diff
@@ -32,20 +32,250 @@
...
+        </div>
+        <ul class="nav-gallery">
+          <li class="circle active-circle"><span class="border"></span></li>
```
Comment (iliatcymbal) [inline index.html]: 1) no need span here, use pseudo-elements 
2) no need class on every `li`
3) use selectors `.gallery .paging li` and `.gallery .paging li.active`
4) on hover should be the same effect as on the active state

File: styles.css [hunk 6]
```diff
@@ -8,8 +8,42 @@
...
+  src: url(fonts/Lato-Regular.ttf);
+}
+
 body {
+  display: flex;
+  flex-direction: column;
   background-color: #111619;
+  min-width: 1900px;
+  font: 16px/24px PlayfairDisplay-Regular, sans-serif;
 }
```
Comment (iliatcymbal) [inline styles.css]: make min-width according to design, remove default margins

File: styles.css [hunk 7]
```diff
@@ -155,23 +209,603 @@ body {
...
+.company-offer figcaption {
+  width: 110px;
+  transform: translate(520px, -157px);
+}
+
+.portfolio {
+  padding: 157px 0 153px;
+  text-align: center;
+  background: url(image/portfolioImages/portfolio.png) no-repeat 45px 2px;
+}
```
Comment (iliatcymbal) [inline styles.css]: image

File: styles.css [hunk 8]
```diff
@@ -155,23 +209,603 @@ body {
...
+
+.portfolio li:hover > .active-button {
+  display: block;
+}
+
+.services {
+  padding: 61px 0 130px;
+  background: url(image/roadPicture.png) no-repeat 0 175px;
+  text-align: center;
+}
```
Comment (iliatcymbal) [inline styles.css]: image

File: styles.css [hunk 9]
```diff
@@ -104,7 +151,8 @@ body {
 }
 
 .gallery {
-  background: url(image/backgroundGallery.png) no-repeat 43px 184px;
+  background: url(image/backgroundGallery.png) no-repeat 51px 184px;
+  padding: 0 0 75px;
 }
```
Comment (iliatcymbal) [inline styles.css]: image
use ctrl -/+ to resize a window to see how it looks on different scales

File: index.html [hunk 10]
```diff
@@ -32,20 +32,250 @@
...
+        <h1>
+          We're a designer & a developer who love building cool stuff with our clients.
+        </h1>
+        <img src="image/painting.png" alt="Painting">
+      </article>
+    </div>
+    <div class="strategy">
+      <div class="content">
+        <figure>
+          <img src="image/productImage.png" alt="Product image">
```
Comment (iliatcymbal) [inline index.html]: `image/productImage01.png`

File: index.html [hunk 11]
```diff
@@ -32,20 +32,250 @@
...
+          <h1>
+            We are bunch of super-skilled<br>guys and gals who love<br>to create websites.
+          </h1>
+          <p>
+            Filema Rodion proposes individual, airtight packages that keep the product's characteristics unchanged for treats for the marriage, baptism and every social event.
+          </p>
+          <a href="#">LEARN MORE</a>
+        </article>
+        <figure>
+          <img src="image/productImageTwo.png" alt="Product image">
```
Comment (iliatcymbal) [inline index.html]: `image/productImage02.png` and so on...

File: index.html [hunk 12]
```diff
@@ -32,20 +32,250 @@
...
+          <li class="end-of-thino">
+            <figure class="active">
+              <img src="image/portfolioImages/vacuumCleaner.png" alt="Vacuum cleaner">
+              <figcaption>
+                <h2>End of Thino</h2>
+                <span>
+                  When the switch is on, it cuts off the data transmission<br>between the charging device and the source
+                </span>
+              </figcaption>
+            </figure>
```
Comment (iliatcymbal) [inline index.html]: 1) you don't need to set up a class on every `li`
2) figure should have fixed height, and image should be centered inside it
3) items shouldn't jump while hovering
4) hover should work only when hover on white part
5) no need figcaption, it is a description of the product

File: index.html [hunk 13]
```diff
@@ -32,20 +32,250 @@
...
         <ul class="nav-gallery">
-          <li class="circle active"></li>
-          <li class="circle"></li>
-          <li class="circle"></li>
-          <li class="circle"></li>
+          <li class="circle active-circle"><span class="border"></span></li>
+          <li class="circle"><span></span></li>
+          <li class="circle"><span></span></li>
+          <li class="circle"><span></span></li>
         </ul>
```
Comment (iliatcymbal) [inline index.html]: see my previous comments for the gallery how to make markup

File: index.html [hunk 14]
```diff
@@ -32,20 +32,250 @@
...
+        </article>
+        <ul class="list-of-services">
+          <li>
+            <img src="image/agilityImage.png" alt="Agility">
+            <h2>Agility</h2>
+            <p>
+              We believe in a seamless client partnership that<br>connects us into a single unified team. We're not only<br>agile in code but in discovery, and design.
+            </p>
+          </li>
+          <li>
```
Comment (iliatcymbal) [inline index.html]: it shouldn't jump on hover

File: styles.css [hunk 15]
```diff
@@ -21,6 +55,12 @@ body {
...
   padding: 0 0 29px;
+  flex: 0 0 auto;
+  width: 100%;
+}
+
+main {
+  flex: 1 0 auto;
 }
 
 .logo-and-contacts {
```
Comment (iliatcymbal) [inline styles.css:59]: a big area under the logo is clickable

File: styles.css [hunk 16]
```diff
@@ -155,23 +209,603 @@ body {
...
+  letter-spacing: 2px;
+}
+
+.strategy a:hover ,
+.company-offer a:hover {
+  text-decoration: underline;
+}
+
+.company-offer {
+  padding: 172px 0 127px;
```
Comment (iliatcymbal) [inline styles.css]: Why do you use paddings but not margins?
The upper element (strategy) should have a bottom margin

File: styles.css [hunk 17]
```diff
@@ -8,8 +8,42 @@
...
+}
+
+@font-face {
+  font-family: "Lato-Regular";
+  src: url(fonts/Lato-Regular.ttf);
+}
+
 body {
+  display: flex;
+  flex-direction: column;
```
Comment (iliatcymbal) [inline styles.css]: I don't see why you need flex on body

File: index.html [hunk 18]
```diff
@@ -32,20 +32,250 @@
...
-          <li class="circle"></li>
+          <li class="circle active-circle"><span class="border"></span></li>
+          <li class="circle"><span></span></li>
+          <li class="circle"><span></span></li>
+          <li class="circle"><span></span></li>
         </ul>
+        <div class="contact">
+          <span>
+            If you'd like to get in touch,<br> you can send ud an email directly at
+            <address>hello@10x.com</address>
```
Comment (iliatcymbal) [inline index.html]: it's not an address, it's an email
```
<a href="mailto:hello@10x.com">hello@10x.com</a>
```

**Review by iliatcymbal — CHANGES_REQUESTED** [review]
1)  `Tube icon`
it is not a tube, it is a phone
 
2) compress images which are more than 10Kb

3) I don't think `<ul class="social-networks">` should be placed inside `nav`

File: styles.css [hunk 19]
```diff
@@ -155,23 +209,603 @@ body {
...
+}
+
+.company-offer {
+  padding: 172px 0 127px;
+}
+
+.company-offer .content {
+  display: flex;
+  column-gap: 108px;
+}
```
Comment (iliatcymbal) [inline styles.css]: Actually I don't see why you need extra block `.company-offer` (or `.content`)

File: index.html [hunk 20]
```diff
@@ -32,20 +32,250 @@
...
+    </div>
+    <div class="music-player">
+      <article class="content">
+        <div class="player">
+          <img src="image/audioPlayer.png" alt="Audio player">
+          <button><img src="image/playButton.png" alt="play button"></button>
+        </div>
+        <h1>
+          We're a designer & a developer who love building cool stuff with our clients.
+        </h1>
```
Comment (iliatcymbal) [inline index.html:87]: This font looks as a bold

File: styles.css (hunk 17 above)
Comment (NikitaBondarevv) [reply styles.css]: 4 Using flexbox https://sgeek.pro/web/coding/5-sposobov-kak-prizhat-footer-k-nizu-stranitsy/

Comment (iliatcymbal) [reply styles.css]: 1) there wasn't task to make footer at bottom
2) it doesn't work

File: styles.css [hunk 21]
```diff
@@ -155,23 +214,540 @@ body {
...
+  align-items: center;
+}
+
+.music-player h1 {
+  text-align: center;
+  font-size: 70px;
+  line-height: 100px;
+  color: #fff;
+  width: 1550px;
+  font-weight: 100;
```
Comment (iliatcymbal) [inline styles.css:267]: usually we use standard aliases
`font-weight: normal/bold/bolder...`

**Pull request merged**